packages/chart/src/plotly/Plot.tsx

```diff
@@ -0,0 +1,15 @@
+import { LoadingOverlay } from '@deephaven/components';
+import { lazy, Suspense } from 'react';
+
+const PlotBase = lazy(() => import('./PlotBase.js'));
```

Comment:
No need for .js.

I'm inexplicably sad you didn't name this `LazyPlot` and use the same naming used for `LazyChart`/`Chart`.

Side note - why _does_ `HeapUsage` use `Plot` directly instead of `Chart`? Were we just being lazy there/didn't want to make a model to pass to `Chart`?

Reply:
I can rename it to match the other naming.

That would be my guess is we didn't want to make a model. Or there's other things `Chart` does that we didn't want in `HeapUsage`. Not sure